nova_vm/src/engine/bytecode/executable.rs

```diff
@@ -551,13 +488,47 @@ impl CompileEvaluation for ast::BinaryExpression<'_> {
             ctx.exe.add_instruction(Instruction::GetValue);
         }
 
-        ctx.exe.add_instruction(Instruction::Load);
-
-        // 5. Return ? ApplyStringOrNumericBinaryOperator(lval, opText, rval).
-        ctx.exe
-            .add_instruction(Instruction::ApplyStringOrNumericBinaryOperator(
-                self.operator,
-            ));
+        match self.operator {
+            BinaryOperator::LessThan => {
```

Comment:
**praise:** Oh yeah <3 I'd been looking at and confused by this code for a good while.